# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../clearsilver/zlib-Makefile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./cgi/Makefile.old	2005-01-12 23:31:25.215403672 +0100
+++ ./cgi/Makefile	2005-01-12 23:32:39.086173608 +0100
@@ -16,7 +16,6 @@
 STATIC_CSO = $(STATIC_EXE:%.cgi=%.cso)
 
 DLIBS += -lneo_cgi -lneo_cs -lneo_utl # -lefence
-LIBS += -L$(LIB_DIR) $(DLIBS)
 
 TARGETS = $(CGI_LIB) $(STATIC_EXE) $(STATIC_CSO)
 
@@ -27,10 +26,10 @@
 	$(RANLIB) $@
 
 $(STATIC_EXE): $(STATIC_OBJ) $(DEP_LIBS)
-	$(LD) $@ $(STATIC_OBJ) $(LIBS)
+	$(LD) $@ $(STATIC_OBJ) -L$(LIB_DIR) $(DLIBS) $(LIBS)
 
 $(STATIC_CSO): $(STATIC_OBJ) $(DEP_LIBS)
-	$(LDSHARED) -o $@ $(STATIC_OBJ) $(LIBS)
+	$(LDSHARED) -o $@ $(STATIC_OBJ) -L$(LIB_DIR) $(DLIBS) $(LIBS)
 
 install: all
 	$(NEOTONIC_ROOT)mkinstalldirs $(DESTDIR)$(cs_includedir)/cgi
